### ISSUE TITLE
in_cpu outputs one time only

### ISSUE THREAD

**Issue by nokute78** [fluent/fluent-bit#103]
`in_cpu` outputs records one time only.
A variable `flush_done` causes this issue.

``` sh
$ bin/fluent-bit -i cpu -o stdout -vvv
Fluent-Bit v0.9.0
Copyright (C) Treasure Data

[2016/08/22 21:15:11] [ info] [engine] started
[2016/08/22 21:15:11] [debug] [router] default match rule cpu.0:stdout.0
[2016/08/22 21:15:12] [trace] [in_cpu] CPU 8.00%
[2016/08/22 21:15:13] [trace] [in_cpu] CPU 11.00%
[2016/08/22 21:15:14] [trace] [in_cpu] CPU 8.00%
[2016/08/22 21:15:15] [trace] [in_cpu] CPU 2.00%
[2016/08/22 21:15:16] [trace] [engine dispatch] task created 0x1d166b0
[2016/08/22 21:15:16] [trace] [thread 0x1d16780] created
[0] cpu.0: [1471868112, {"cpu_p"=>8.000000, "user_p"=>2.000000, "system_p"=>6.000000, "cpu0.p_cpu"=>8.000000, "cpu0.p_user"=>2.000000, "cpu0.p_system"=>6.000000}]
[1] cpu.0: [1471868113, {"cpu_p"=>11.000000, "user_p"=>4.000000, "system_p"=>7.000000, "cpu0.p_cpu"=>11.000000, "cpu0.p_user"=>4.000000, "cpu0.p_system"=>7.000000}]
[2] cpu.0: [1471868114, {"cpu_p"=>8.000000, "user_p"=>1.000000, "system_p"=>7.000000, "cpu0.p_cpu"=>8.000000, "cpu0.p_user"=>1.000000, "cpu0.p_system"=>7.000000}]
[3] cpu.0: [1471868115, {"cpu_p"=>2.000000, "user_p"=>1.000000, "system_p"=>1.000000, "cpu0.p_cpu"=>2.000000, "cpu0.p_user"=>1.000000, "cpu0.p_system"=>1.000000}]
[2016/08/22 21:15:16] [trace] [engine] [task event] task_id=0 thread_id=0 return=OK
[2016/08/22 21:15:16] [trace] [thread] destroy thread_id=0
[2016/08/22 21:15:16] [trace] [engine] destroy task_id=0

(no output)
```

**Comment by edsiper** [comment]
oooops, I merged some code that should not be there.

**Comment by edsiper** [comment]
fixed by 8e77ef48a086aec40dc3ac7a5f3e0c5d0e59a326

**Comment by nokute78** [comment]
Thank you for fixing!
